app/assets/stylesheets/custom.scss

```diff
@@ -24,6 +24,10 @@ h6 {
   font-weight: 400;
 }
 
+table a, a:hover{
```

Comment:
Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space